app/(afterLogin)/(okrplanning)/okr/dashboard/_components/AwaitingApprovalsList.tsx

```
@@ -36,7 +36,10 @@ const AwaitingApprovalsList: React.FC = () => {
   // Check if user has subordinates
   const subordinates =
     employeeData?.items?.filter(
-      (employee: any) => employee.reportingTo?.id === userId,
+      (employee: any) =>
+        employee?.delegatedTo?.id
```

ESLint

[error] Delete `··`

(prettier/prettier)